src/community/resources-and-tools/index.md

```diff
@@ -79,6 +78,11 @@ GOV.UK Frontend WTForms Widgets.
 
 [GOV.UK Frontend Flask](https://github.com/LandRegistry/govuk-frontend-flask) - Complete Flask app template that implements the Jinja and WTForms packages to integrate with GOV.UK Frontend.
 
+### PHP / Laravel
+
+[GOV.UK Laravel](https://github.com/AnthonyEdmonds/govuk-laravel) -
+A collection of GOV.UK Frontend Laravel blade components, validation rules, and page building templates and helpers.
```

Comment:
Brill, thank you. Definitely better with more info 🎉

Can I double check, do we need to title case [Blade](https://laravel.com/docs/10.x/blade#introduction) as a product name?

Perhaps a little fix for the double "and" then we're good to go?

>…, **_and_** page building templates **_and_** helpers.


```suggestion
A collection of GOV.UK Frontend Laravel Blade components and validation rules, with page building templates and helpers.
```